src/twisted/test/test_twistd.py

```diff
@@ -1014,6 +1013,11 @@ def test_setUidSameAsCurrentUid(self):
         If the specified UID is the same as the current UID of the process,
         then a warning is displayed.
         """
+
+        # Assert that there were no existing warnings.
+        existing_warnings = self.flushWarnings()
+        self.assertEqual([], existing_warnings)
+
```

Comment:
Maybe add a link to the issue so that we will review this test once the test suite has some hooks for warnings.

```suggestion
        # FIXME:https://github.com/twisted/twisted/issues/10332
        # Assert that there were no existing warnings.
        existing_warnings = self.flushWarnings()
        self.assertEqual([], existing_warnings)

```